rs/backend/src/tvl.rs

```diff
@@ -1,18 +1,30 @@
 use crate::{
     canisters::{exchange_rate_canister, governance},
-    constants::NANOS_PER_UNIT,
+    constants::{E8S_PER_UNIT, NANOS_PER_UNIT},
     spawn,
     state::STATE,
     time,
     timer::{set_timer, set_timer_interval},
 };
+use candid::{CandidType, Nat};
 use std::time::Duration;
 
 pub mod state;
 
 const XRC_MARGIN_SECONDS: u64 = 60 * 5;
 const UPDATE_INTERVAL_SECONDS: u64 = 6 * 60 * 60; // 4 times a day
 
+#[derive(CandidType, Debug, PartialEq)]
+pub struct TvlResult {
+    pub tvl: Nat, // Total Value Locked in whole USD.
```

Comment:
NCR; just for your consideration

```suggestion
    pub tvl_usd: Nat,
```

Reply:
I know this is not ideal, which is why I wrote in the PR description:
> The return type is the way it is to make it compatible with the existing TVL canister. This allows us to migrate the frontend from using the TVL canister to getting the same data from the nns-dapp canister without changing frontend code.

So this has to be compatible with https://github.com/dfinity/ic/blob/cd9216dda55554ff4c83dbac41b7ccb9dc6599d1/rs/rosetta-api/tvl/src/types.rs#L30-L33

After migrating, we could change it, although I think it's probably not worth the effort.